src/pages/javascript/tutorials/what-does-void-0-mean/index.md

```diff
@@ -1,9 +1,9 @@
 ---
-title: What Does JavaScript Void 0 Mean
+title: What Does Void 0 Mean
```

Comment:
I think this title should remain the same. This is talking about links with `href="javascript:void(0)"`. Maybe the title should be changed to "What javascript:void(0) means"